lbrynet/daemon/Daemon.py

```diff
@@ -3082,29 +3045,264 @@ def jsonrpc_stream_availability(self, uri, search_timeout=None, blob_timeout=Non
+    def jsonrpc_balance(self, account_name=None, confirmations=6, include_reserved=False,
+                        include_claims=False):
+        """
+        Return the balance of an individual account or all of the accounts.
+
+        Usage:
+            balance [<account_name>] [--confirmations=<confirmations>]
+                    [--include_reserved] [--include_claims]
+
+        Options:
+            --account=<account_name>        : (str) If provided only the balance for this
+                                                    account will be given
+            --confirmations=<confirmations> : (int) required confirmations (default: 6)
+            --include_reserved              : (bool) include reserved UTXOs (default: false)
+            --include_claims                : (bool) include claims, requires than a
+                                                     LBC account is specified (default: false)
+
+        Returns:
+            (map) balance of account(s)
+        """
+        if account_name:
+            for account in self.wallet.accounts:
+                if account.name == account_name:
+                    if include_claims and not isinstance(account, LBCAccount):
+                        raise Exception(
+                            "'--include-claims' requires specifying an LBC ledger account. "
+                            "Found '{}', but it's an {} ledger account."
+                            .format(account_name, account.ledger.symbol)
+                        )
+                    args = {
+                        'confirmations': confirmations,
+                        'include_reserved': include_reserved
+                    }
+                    if include_claims:
+                        args['include_claims'] = True
+                    return account.get_balance(**args)
+            raise Exception("Couldn't find an account named: '{}'.".format(account_name))
+        else:
+            if include_claims:
+                raise Exception("'--include-claims' requires specifying an LBC account.")
+            return self.wallet.get_balances(confirmations)
+
+    @requires("wallet")
+    def jsonrpc_max_address_gap(self, account_name):
+        """
+        Finds ranges of consecutive addresses that are unused and returns the length
+        of the longest such range: for change and receiving address chains. This is
+        useful to figure out ideal values to set for 'receiving_gap' and 'change_gap'
+        account settings.
+
+        Usage:
+            max_address_gap (<account_name> | --account=<account_name>)
+
+        Options:
+            --account=<account_name>        : (str) account for which to get max gaps
+
+        Returns:
+            (map) maximum gap for change and receiving addresses
+        """
+        return self.get_account_or_error('account', account_name).get_max_gap()
+
+    @requires("wallet")
+    def jsonrpc_fund(self, to_account, from_account, amount=0,
```

Comment:
`wallet_fund`

Reply:
In what context would `fund` mean something other than wallet related funding operations?

Reply:
An `Index fund` :-P